code/workspaces/web-app/src/components/notebooks/NotebookSharingOptions.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  select: {
+    whiteSpace: 'normal',
+    maxWidth: 500,
+  },
+  warning: {
+    color: 'hsl(0, 40%, 40%)',
+  },
+}));
```

Comment:
I think there needs to be more styling on these components. Currently there is an overall lack of spacing. I think all of the pieces of descriptive text need to have a `marginTop` of `theme.spacing(1)`. I also think the `select` class needs to have some extra padding, potentially `theme.spacing(1)` top and bottom and nothing extra on the sides.

I think the name of the status level is also a bit lost at the moment and would benefit from having a `fontWeight` of `500`.